packages/developer-extension/src/background/domain/replaceBundles.ts

```diff
@@ -0,0 +1,51 @@
+import { listenAction } from '../actions'
+import { DEV_LOGS_URL, DEV_RUM_RECORDER_URL, DEV_RUM_URL } from '../constants'
+import { setStore, store } from '../store'
+
+chrome.webRequest.onBeforeRequest.addListener(
```

Comment:
This does not work well when bundles are cached by browser.
what about either:
- adapt the strategy to work even when bundles are cached
- add a "full reload may be needed" on the UI

Reply:
I'll see what I can do, it seems that there is [an API for it](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/removeCache)